app/grants/management/commands/payout_round_noncustodial.py

```diff
@@ -168,14 +172,16 @@ def handle(self, *args, **options):
                     ready_for_test_payout=ready_for_test_payout,
                     grant_payout=grant_payout
                 )
+                clr_matches_created += 1
                 if needs_kyc:
                     grant_match_distribution_kyc(match)
+            self.stdout.write(f"{clr_matches_created} matches were created")
 
         # Payout rankings (round must be finalized first) ------------------------------------------
         if what in ['prepare_final_payout']:
             payout_matches = scheduled_matches.filter(ready_for_payout=False)
             payout_matches_amount = sum(sm.amount for sm in payout_matches)
-            print(f"there are {payout_matches.count()} UNPAID Match Payments already created worth ${round(payout_matches_amount,2)} {network} DAI")
+            self.stdout.write(f"there are {payout_matches.count()} UNPAID Match Payments already created worth ${round(payout_matches_amount,2)} {network} DAI")
             print(f"All these are users would need KYC. Do not do this for mainnet")
             print('------------------------------')
```

Comment:
Should all the print statements be converted to use `self.stdout.write`? There are a few below as well